spec/01-unit/03-prefix_handler_spec.lua

```diff
@@ -126,10 +126,17 @@ describe("NGINX conf compiler", function()
   describe("compile_nginx_conf()", function()
     it("compiles a main NGINX conf", function()
       local nginx_conf = prefix_handler.compile_nginx_conf(helpers.test_conf)
-      assert.matches("user nobody nobody;", nginx_conf, nil, true)
+      assert.not_matches("user nobody nobody;", nginx_conf, nil, true)
```

Comment:
@thibaultcha, It was merged 6 days ago here by you ;-):
https://github.com/depay/kong/commit/f3572acf17eacd765610a2f41fb288582fea3c47#diff-eba096d79e40175ecfdc56a83c5e4e6d